tests/helpers.py

```diff
@@ -240,9 +267,14 @@ def create_pvc(sc_name, pvc_name=None):
     )
     pvc_data['metadata']['namespace'] = defaults.ROOK_CLUSTER_NAMESPACE
     pvc_data['spec']['storageClassName'] = sc_name
-    return create_resource(
-        desired_status=constants.STATUS_BOUND, **pvc_data
-    )
+    if size:
+        pvc_data['spec']['resources']['requests']['storage'] = size
+    ocs_obj = pvc.PVC(**pvc_data)
+    ocs_obj.create()
+    ocs_obj.reload()
```

Comment:
why is this needed? create() does reload()

Reply:
Yes, but 'backed_pv' was still None cus it takes little time to update 'pv' once pvc created as I have observed.

Reply:
so do the reload() only if you pass wait=True.
If you don't want to wait this second call to reload() will take valuable time when we'll do disruptive operations